examples/golang/object-upload.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+    "github.com/aws/aws-sdk-go/aws/credentials"
```

Comment:
can  you please run the code through https://go.dev/blog/gofmt
make sure that indentation and formatting is good

Reply:
Thanks Yuval for that
already done it
kindly have a look